tests/e2e/creation_test.go

```diff
@@ -37,7 +37,7 @@ func creationTestSuite(t *testing.T) {
 			err = testCtx.testDSCICreation()
 			require.NoError(t, err, "error creating DSCI CR")
 		})
-		if testCtx.testOpts.webhookTest {
+		if testOpts.webhookTest {
```

Comment:
I would prefer to keep options as a part of context, instead of access global variable

Reply:
yeah, however there is dependency cycle here, because the scenarios maps to methods that are invoking `NewTestContext`, which depends on the global opts struct.

if you don't mind, I'll try to explore a few options as follow up